CONTRIBUTION.md

```diff
@@ -0,0 +1,39 @@
+Branching names:
+---
+Each branch must have a name in the following format with a proper prefix and number of Github's ticket it refers to:
+
+```
+prefix/000-branch-name
+``` 
+
+Allowed prefixes:
+* feature
+* bugfix
+* docs
+* test (when adding missing tests)
+* ...
```

Comment:
+improvement?

Reply:
`refactor` sounds ok to me

also I usually find that `chores` can turn out useful, for those tasks that needs be done for infrastructure compliance, some policy change, ...